pkg/river/vm/vm_block_test.go

```diff
@@ -78,7 +78,7 @@ func TestVM_Block_Attributes(t *testing.T) {
 		eval := vm.New(parseBlock(t, input))
 
 		err := eval.Evaluate(nil, &block{})
-		require.EqualError(t, err, `"number" must be an attribute, but is used as a block`)
+		require.EqualError(t, err, `2:4: "number" must be an attribute, but is used as a block`)
```

Comment:
This error (and the one below it) is a diag.Diagnostic now, so I had to add the position information.